eva-server/src/main/java/uk/ac/ebi/eva/server/ws/GeneWSServerV2.java

```diff
@@ -59,13 +60,23 @@ public GeneWSServerV2() {
     }
 
     @GetMapping(value = "/{geneIds}/variants")
-    public ResponseEntity getVariantsByGene(@PathVariable("geneIds") List<String> geneIds,
-                                            @RequestParam(name = "species") String species,
-                                            @RequestParam(name = "assembly") String assembly,
-                                            @RequestParam(required = false, defaultValue = "0") Integer pageNumber,
-                                            @RequestParam(required = false, defaultValue = "20") Integer pageSize,
-                                            HttpServletResponse response,
-                                            @ApiIgnore HttpServletRequest request)
+    public ResponseEntity getVariantsByGene(
+            @ApiParam(value = "Comma separated values of gene names or gene ids. e.g. BRCA2,FOXP2,ENSG00000223972")
+            @PathVariable("geneIds") List<String> geneIds,
+            @ApiParam(value = "First letter of the genus, followed by the full species name, e.g. hsapiens. Allowed" +
```

Comment:
I suggest that "hsapiens" and "grch37" are made examples for illustration purposes of a default search.